<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpperCarWidget</class>
 <widget class="QWidget" name="UpperCarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>‰∏≤Âè£ËÆæÁΩÆ</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4,0,0,1,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ËÆæÂ§áÔºö</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_devices"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_refresh">
        <property name="text">
         <string>Âà∑Êñ∞üîÑÔ∏è</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ê≥¢ÁâπÁéáÔºö</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_bandrate">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>ËøûÊé•üîó</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>ËÆæÁΩÆÊéßÂà∂</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_turn_left">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÈÄÜÊó∂Èíà</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QSlider" name="slider_forward_speed">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_forward">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÂâçËøõ</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_move_left">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Â∑¶ÊºÇ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ÊåáÂÆöÈÄüÂ∫¶ÂâçËøõÔºö</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_backward">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÂêéÈÄÄ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_turn_right">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>È°∫Êó∂Èíà</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btn_tracker_start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÂºÄÂêØÂ∑°Á∫ø</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_light">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ËΩ¶ÁÅØ</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btn_tracker_stop">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÂÖ≥Èó≠Â∑°Á∫ø</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btn_move_right">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Âè≥ÊºÇ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btn_buzzer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ËúÇÈ∏£Âô®</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QPushButton" name="btn_stop">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>ÂÅúËΩ¶</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_forward_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_forward_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
